Specs/DateTime/Spanish/DatePeriodExtractor.json

```diff
@@ -540,23 +415,27 @@
   },
   {
     "Input": "Estare afuera la tercer semana de 2027",
+    "NotSupported": "dotnet",
+    "NotSupportedByDesign": "javascript, python, java",
     "Results": [
       {
-        "Text": "la tercer semana de 2027",
+        "Text": "tercer semana de 2027",
         "Type": "daterange",
-        "Start": 14,
-        "Length": 24
+        "Start": 17,
+        "Length": 21
       }
     ]
   },
   {
     "Input": "Estare afuera la tercer semana proximo año",
+    "NotSupported": "dotnet",
+    "NotSupportedByDesign": "javascript, python, java",
     "Results": [
       {
-        "Text": "la tercer semana proximo año",
+        "Text": "tercer semana proximo año",
```

Comment:
In English, this test resolves `the third week next year` from the input `I'll be out the third week next year` maintaining `the` keyword, so in Spanish, we should keep `la` equivalent.